docs/root/start/install.rst

```diff
@@ -13,48 +13,47 @@ getting your Envoy proxy up and running.
 Install Envoy on Debian GNU/Linux
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You can `install Envoy on Debian <https://www.getenvoy.io/install/envoy/debian/>`_
-using `Get Envoy <https://www.getenvoy.io/>`__ until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
+You can `install Envoy on Debian using packages created by Tetrate <https://cloudsmith.io/~tetrate/repos/getenvoy-deb-stable/setup/#formats-deb>`_
+until `official packages exist <https://github.com/envoyproxy/envoy/issues/16867>`_.
 
 .. code-block:: console
 
-   $ sudo apt update
-   $ sudo apt install apt-transport-https ca-certificates curl gnupg2 software-properties-common
-   $ curl -sL 'https://getenvoy.io/gpg' | sudo gpg --dearmor -o /usr/share/keyrings/getenvoy-keyring.gpg
-   # Verify the keyring - this should yield "OK"
-   $ echo 1a2f6152efc6cc39e384fb869cdf3cc3e4e1ac68f4ad8f8f114a7c58bb0bea01 /usr/share/keyrings/getenvoy-keyring.gpg | sha256sum --check
-   $ echo "deb [arch=amd64 signed-by=/usr/share/keyrings/getenvoy-keyring.gpg] https://dl.bintray.com/tetrate/getenvoy-deb $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/getenvoy.list
-   $ sudo apt update
-   $ sudo apt install getenvoy-envoy
+   $ apt-get update
```

Comment:
can we keep the sudo - also minor nit but for enduser use i tend to use apt over apt-get

Reply:
I removed sudo because it breaks the instructions on the official docker images as it isn't in the path by default. I generally don't like adding instructions that I can't easily verify or require additional steps to operate, but if you feel strongly, I can add it back.

```bash
$ docker run -ti --rm debian
root@2c6dceaf2d6e:/# sudo 
bash: sudo: command not found

$ docker run -ti --rm ubuntu
root@4c41aadc0e0d:/# sudo
bash: sudo: command not found
```

Reply:
hmm - so in distros rather than docker images - ubuntu has sudo preinstalled, but debian doesnt

its the ubuntu users im more concerned with tbh - and as a long-time debian user i have reasonable confidence that most debian users can work it out either way

Reply:
as both sides of this are subjective and the current content is verified. do you mind merging this? I think it is causing more damage and probably we won't hear anyone complain either from lack or presence of sudo. usually nits like this especially subjective ones won't block progress.. let's get this in?